extensions/grpc-common/runtime/pom.xml

```diff
@@ -99,6 +99,24 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-vertx</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.grpc</groupId>
```

Comment:
How come we only now need this?!

Reply:
They moved classes from the API module to the core module. It's annoying, as we are not depending on implementation details. We would need to revisit that at some point.